runtime/error.c

```diff
@@ -1,9 +1,116 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define MIN_FRAME_COUNT 2
+#define SKIP_FROM_END 1 // This number counts from the last stack frame
+#define THREAD 0
+
+typedef struct {
+    uint32_t nFrames;
+    uint32_t capacityOfFrames;
+    uint32_t nBytes;
+    uint32_t capacityOfBytes;
+    Frame frames[];
+} FramesInfo;
+
+void on_error(void *vdata, const char *msg, int errnum) { printf("Error : %s\n", msg); }
```

Comment:
Why do we need this?

Reply:
This is the implementation of libbacktrace error callback function. I thought that it will be useful to catch libbacktrace related errors.